config/development.json

```diff
@@ -65,7 +65,7 @@
 		"external-media/openverse": true,
 		"fediverse/allow-opt-in": true,
 		"full-site-editing/beta-opt-in": true,
-		"cookie-banner": false,
+		"cookie-banner": true,
```

Comment:
I think this sneaked in from testing :)

Reply:
Thanks for mentioning it @gmovr !  🙌 
I have removed it form the pr